container/sagemaker/proxy_client.py

```diff
@@ -73,6 +74,36 @@ def add_model(self, model_name, base_path, model_platform='tensorflow'):
 
         return 'Successfully loaded model {}'.format(model_name)
 
+    def delete_model(self, model_name):
+        model_server_config = model_server_config_pb2.ModelServerConfig()
+
+        with lock(DEFAULT_LOCK_FILE):
+            try:
+                config_file = self._read_model_config(MODEL_CONFIG_FILE)
+                model_server_config = text_format.Parse(text=config_file,
+                                                        message=model_server_config)
+
+                config_json = json_format.MessageToJson(model_server_config)
+                config_json = json.loads(config_json)
+
+                if config_json['modelConfigList']:
+                    config_json['modelConfigList']['config'] = [c for c in config_json['modelConfigList']['config']
+                                                                if model_name != c['name']]
+                    model_server_config = json_format.Parse(json.dumps(config_json), message=model_server_config)
```

Comment:
model_server_config and config_proto look like (near) duplicates. why do we need both?

Reply:
Yeah, they are duplicate, I've removed the config_proto.